logs_ingest/main.py

```diff
@@ -148,3 +151,8 @@ def parse_record(record: Dict, self_monitoring: SelfMonitoring):
             parsed_record["content"] = parsed_record["content"][
                                        :trimmed_len] + DYNATRACE_LOG_INGEST_CONTENT_MARK_TRIMMED
     return parsed_record
+
+
+def extract_cloud_log_forwarder(parsed_record):
+    if function_app_name:
+        parsed_record["cloud.log_forwarder"] = parsed_record[RESOURCE_ID_ATTRIBUTE] + ": " + function_app_name
```

Comment:
Now **cloud.log_forwarder** attribute contains **full resource_id**   and **Function App name** at the end.
I hope I understand right finally :slightly_smiling_face: